fhir/com.b2international.snowowl.fhir.core/src/org/linuxforhealth/fhir/core/ResourceType.java

```diff
@@ -0,0 +1,1775 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.linuxforhealth.fhir.core;
+
+import static org.linuxforhealth.fhir.core.FHIRVersionParam.VERSION_40;
+import static org.linuxforhealth.fhir.core.FHIRVersionParam.VERSION_43;
+import static org.linuxforhealth.fhir.core.FHIRVersionParam.VERSION_50;
+
+/*
+ * Modifications:
+ * 
+ * - Added literals for resource types appearing in FHIR R5
+ */
+
+/**
+ * Enum constants for all resource types across all versions of HL7 FHIR
+ */
+public enum ResourceType {
```

Comment:
Please remove the ones that we do not support.

Reply:
The XML parser recognizes elements from the spec via this enum (and can then say that it does not support deserializing them). If I remove the literals, they'll turn into invalid types instead of unsupported: https://github.com/b2ihealthcare/snow-owl/blob/89d4ab735d72832cbe7ced57175fc45f80ea894d/fhir/com.b2international.snowowl.fhir.core/src/org/linuxforhealth/fhir/model/r5/parser/FHIRXMLParser.java#L8889

Reply:
Hmm, my problem is the size of this enum 😄 but okay, leave it like this for now.